clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java

```diff
@@ -372,7 +371,7 @@ private void updateReconnectBackoff(NodeConnectionState nodeState) {
     /**
      * Increment the failure counter and update the node connection setup timeout exponentially.
      * The delay is socket.connection.setup.timeout.ms * 2**(failures) * (+/- 20% random jitter)
-     * Up to a (pre-jitter) maximum of reconnect.backoff.max.ms
+     * Up to a (pre-jitter) maximum of socket.connection.setup.timeout.max.ms
```

Comment:
side fix: the java doc is talking about `socket.connection.setup.timeout.ms`, not `reconnect.backoff.ms`.